modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -163,3 +170,72 @@ image::xdcr/filter-replication-diagram-7.png[,720,align=left]
 
 The new document `airline_8838` is added the source bucket, and is examined by _R1a_.
 A successful match is made, and `airline_8838` is duly replicated to the target bucket.
+
+[#using-deletion-filters]
+== Using Deletion Filters
+
+_Deletion Filters_ control whether the deletion of a document at source causes the deletion of a corresponding replica-document on the target.
+These filters are individually described in xref:manage:manage-xdcr/filter-xdcr-replication.adoc#deletion-filters[Deletion Filters].
+
+By default, any source-document deletion _is_ replicated to the target; resulting in a corresponding target-document deletion.
+Note that such replication is _not_ prevented by the specifying of a filter that is formed with regular and other filtering expressions: such expressions only determine which non-deleted documents are to be replicated.
+Therefore, to ensure that document-deletions are _not_ replicated, _deletion filters_ must specifically be configured.
+
+=== Configuring Deletion-Filters to Protect Data
+
+Appropriate deletion-filter settings protect data.
+However, in certain circumstances, inappropriate deletion-filter settings may cause _loss_ of data.
```

Comment:
This example talks about the loss of data. There is also situations where an inappropriate filter setting may cause accidental leak of stale data.

For example, if the user turns on the deletion filter in a replication, and the situation is where source bucket contains documents that contains sensitive information. If a source document was deleted for compliance purposes, then the target document could become stale and remain, even if were not meant to exist anymore.

Reply:
I've added a similar subsection, which hopefully addresses this.